packages/config/config/devices/0x001d/dzpd3.json

```diff
@@ -123,5 +136,10 @@
 				}
 			]
 		}
+	},
+	"metadata": {
+		"inclusion": "Network Wide Inclusion:\n1. Network Wide Inclusion allows your device to be added to the network using devices already in the network to assist with communication. Work your way from the closest devices to the controller outward.\n2. • Enter Programming Mode by holding the control button for 7 seconds, the Locator LED will blink amber.\n3. • Tap the control button one time. The Locator LED will quickly flash green.\n4. • The Decora Smart Z-Wave device is ready to learn into the Z-Wave network.\n5. • Follow directions in the Z-Wave controller to complete the adding process.\n6. • Upon successful addition to network the LED will turn off and then blink green 3 times.\n7. • If the adding process is not successful the LED will flash red 3 times.\n8. Traditional Inclusion:\n9. For older controllers Traditional Inclusion is supported. Depending on the age of the controller, the controller will need to be 3 to 35 feet from the device when including.\n10. • Enter Programming Mode by holding the control button for 7 seconds; the Locator LED will blink amber.\n11. • The Decora Smart Z-Wave device is ready to add to the Z-Wave network.\n12. • Follow directions in the Z-Wave controller to enter learn mode.\n13. • Tap the control button one time. The Locator LED will quickly flash green. The Z-Wave controller will begin to pair with the Decora Smart device.\n14. • Upon successful addition to the network the LED will turn off and then blink green 3 times.\n15. • If the adding process is not successful the LED will flash red 3 times.",
+		"exclusion": "• Enter Programming Mode by holding the control button for 7 seconds; the Locator LED will turn amber.\n1. • Follow directions in the Z-Wave controller to enter exclusion mode.\n2. • Tap the control button one time. The Locator LED will quickly flash green. The Z-Wave controller will exclude the Decora Smart device.\n3. • The Z-Wave controller will confirm successful exclusion from the network.",
```

Comment:
These seem to use both 1./2./3. for enumeration as well as bullet points. Please choose one.

Reply:
Done.